static/scripts/favoritesandvotes/favorites.js

```diff
@@ -23,9 +23,10 @@ let addFavorite = (trigger, type) => {
         .then(() => {
             let allTags = document.getElementsByClassName(type.split("_")[1])
             if (type.split("_")[1] == "cocktails") {
-                parentElement.innerHTML = `<i class="fas ${icon}" data-id="${trigger.dataset.id}"
-                                data-user="${trigger.dataset.user}" data-name="${trigger.dataset.name}"
-                                onclick="removeFavorite(this, '${type}')" ></i>`;
+                parentElement.innerHTML = 
+                `<i class="fas ${icon}" data-id="${trigger.dataset.id}"` +
+                                `data-user="${trigger.dataset.user}" data-name="${trigger.dataset.name}"` +
+                                `onclick="removeFavorite(this, '${type}')" ></i>`;
             } else {
                 for (i = 0; i < allTags.length; i++) {
```

Comment:
variable `i` implicitly declared. There are a good number of variables like this in your code...